### PR TITLE
AWS Unity Catalog integration

### PR DESCRIPTION
**Pull request by rportilla-databricks**
- Updated README for main project, Unity Catalog setup
- Add Unity Catalog integration module
- Add industry folder for industry quickstarts

### CODE REVIEW

**Comment by rportilla-databricks** [comment]
@nkvuong , if you can take a look, we'd appreciate it! We added a UC module for both AWS and Azure, updated the README and also provided an end-to-end composition of the modules which will appear in the terraform registry.

**Comment by nkvuong** [comment]
@rportilla-databricks I've run some linting, replaced the azapi resource with azurerm. Also left a comment around how providers in module should be provided.

Happy to make some more changes and fully run the different test examples, just let me know how it should be configured

File: modules/aws_full_governed_ws/versions.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      version = "~>1.6.5"
+    }
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~>4.35.0"
+    }
+  }
+}
+
+provider "aws" {
```
Comment (nkvuong) [inline modules/aws_full_governed_ws/versions.tf:14]: shouldn't have `provider` blocks within a module - https://developer.hashicorp.com/terraform/language/modules/develop/providers

File: modules/azure_uc/providers.tf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+provider "azurerm" {
```
Comment (nkvuong) [inline modules/azure_uc/providers.tf:1]: shouldn't have `provider` blocks within a module - https://developer.hashicorp.com/terraform/language/modules/develop/providers

File: examples/test_aws_full_governed_ws/init.tf [hunk 3]
```diff
@@ -0,0 +1,29 @@
+terraform {
+  backend "s3" {
+    bucket = "databricks-terraform-blueprints2"
+    key    = "test_aws_full_governed_ws.tfstate"
+    region = "us-east-1"
+  }
+  required_providers {
+    databricks = {
+      source  = "databricks/databricks"
+      version = "~>1.6.5"
```
Comment (nkvuong) [inline examples/test_aws_full_governed_ws/init.tf:10]: I've updated all the `required_providers` block to the latest version, let's just make sure the tests still pass (and upgrade the `.lock.hcl` with `terraform init --upgrade`

**Review by nkvuong — CHANGES_REQUESTED** [review]

File: modules/azure_uc/providers.tf (hunk 2 above)
Comment (rportilla-databricks) [reply modules/azure_uc/providers.tf:1]: @nathanknox , can you comment on this? If we don't provide it here, I think we don't define anything in the file so I can delete it. Sound right?

Comment (nathanknox) [reply modules/azure_uc/providers.tf:1]: Yes, this should go away. I think I copied it in by accident or was testing at one point. Should have caught it myself, but missed it.

File: examples/test_aws_full_governed_ws/init.tf (hunk 3 above)
Comment (rportilla-databricks) [reply examples/test_aws_full_governed_ws/init.tf:10]: Thanks @nkvuong , all updates made. Please check and let me know if anything else is pending.

File: modules/azure_uc/providers.tf (hunk 2 above)
Comment (rportilla-databricks) [reply modules/azure_uc/providers.tf]: I removed it.

**Review by nkvuong — APPROVED** [review]
LGTM

**Pull request merged**